x/ibc/core/02-client/handler.go

```diff
@@ -8,63 +8,6 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/ibc/core/02-client/types"
 )
 
-// HandleMsgCreateClient defines the sdk.Handler for MsgCreateClient
-func HandleMsgCreateClient(ctx sdk.Context, k keeper.Keeper, msg *types.MsgCreateClient) (*sdk.Result, error) {
-	clientState, err := types.UnpackClientState(msg.ClientState)
-	if err != nil {
-		return nil, err
-	}
-
-	consensusState, err := types.UnpackConsensusState(msg.ConsensusState)
-	if err != nil {
-		return nil, err
-	}
-
-	if err = k.CreateClient(ctx, msg.ClientId, clientState, consensusState); err != nil {
-		return nil, err
-	}
-
-	ctx.EventManager().EmitEvents(sdk.Events{
-		sdk.NewEvent(
-			types.EventTypeCreateClient,
-			sdk.NewAttribute(types.AttributeKeyClientID, msg.ClientId),
-			sdk.NewAttribute(types.AttributeKeyClientType, clientState.ClientType()),
-			sdk.NewAttribute(types.AttributeKeyConsensusHeight, clientState.GetLatestHeight().String()),
-		),
-		sdk.NewEvent(
-			sdk.EventTypeMessage,
-			sdk.NewAttribute(sdk.AttributeKeyModule, types.AttributeValueCategory),
-		),
-	})
-
-	return &sdk.Result{
-		Events: ctx.EventManager().Events().ToABCIEvents(),
-	}, nil
-}
-
-// HandleMsgUpdateClient defines the sdk.Handler for MsgUpdateClient
-func HandleMsgUpdateClient(ctx sdk.Context, k keeper.Keeper, msg *types.MsgUpdateClient) (*sdk.Result, error) {
-	header, err := types.UnpackHeader(msg.Header)
-	if err != nil {
-		return nil, err
-	}
-
-	if err = k.UpdateClient(ctx, msg.ClientId, header); err != nil {
-		return nil, err
-	}
-
-	ctx.EventManager().EmitEvent(
-		sdk.NewEvent(
-			sdk.EventTypeMessage,
-			sdk.NewAttribute(sdk.AttributeKeyModule, types.AttributeValueCategory),
-		),
-	)
-
-	return &sdk.Result{
-		Events: ctx.EventManager().Events().ToABCIEvents(),
-	}, nil
-}
-
 // HandleMsgUpgradeClient defines the sdk.Handler for MsgUpgradeClient
 func HandleMsgUpgradeClient(ctx sdk.Context, k keeper.Keeper, msg *types.MsgUpgradeClient) (*sdk.Result, error) {
```

Comment:
Why was this not migrated?

Reply:
It has been migrated, but there is a test using this method. That's why I didn't erase this code. Will update the test to use new handler and remove this code.